.claude/commands/embrace.md

```
@@ -140,24 +140,24 @@ Scope: [answer]  Focus: [answer]  Autonomy: [answer]
 
 ### Phase 1 — Discover
 
-Invoke the discover skill:
-```
-Skill(skill: "octo:discover", args: "<user's prompt>")
+Run the discover phase via orchestrate.sh:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)